.gitignore

```diff
@@ -1,54 +1 @@
-# Object files
```

Comment:
I see you removed all the lines in .gitignore, I think we should not modify and change it here.